galaxy.template.yml

```diff
@@ -23,3 +23,5 @@ issues: "https://github.com/EnterpriseDB/edb-ansible/issues"
 dependencies:
   "community.postgresql": "*"
```

Comment:
Do we want to be explicit about the version number for these collections?

Reply:
Fixed